manifests/base/_mesh_overlay-delete-oauth-config-patch.yaml

```diff
@@ -0,0 +1,6 @@
+# DO NOT DELETE. Check kustomization.yaml comments in overlays/service-mesh
+$patch: delete
```

Comment:
mmm not 100% sure, but if we delete the `dashboard-oauth-config`, should we do something with the volume `dashboard-oauth-config-generated`?

Reply:
yes, we handle this in the patches to the deployment https://github.com/opendatahub-io/odh-dashboard/blob/2d2244cfb67a566504530a66b767aa540de9aca4/manifests/overlays/service-mesh/patches/deployment-patch.yaml#L5

Reply:
Would we not want this in your overlay? This is in the base? I'll admit I'm kinda a noob when it comes to the power of the kustomization, but it seems odd to have this in the base for everything.

I'm confused why there are two of these -- one here & one in the overlay 🤔

Reply:
Unfortunately this is due to the weakness/limitation/bug of kustomize - not the power. From comment in overlays/service-mesh/kustomization.yaml
```
# when using bases:
# - ../../base
# the entire directory is truncated and the lookup is performed in base/
# for this reason the files must be there as well. with the same content, as they will be in fact used
```

Reply:
This comment is not accurate for v2 operator - we can safely remove this. Will do this during/after our call :+1: